convert2rhel/actions/__init__.py

```diff
@@ -423,26 +499,13 @@ def resolve_action_order(potential_actions, previously_resolved_actions=None):
     """
     Order the Actions according to the order in which they need to run.
 
-    :param potential_actions: Sequence of Actions which we need to find the
-        order of.
-    :type potential_actions: Sequence
-    :param previously_resolved_actions: Sequence of Actions which have already
-        been resolved into dependency order.
-    :type previously_resolved_actions: Sequence
-    :raises DependencyError: when it is impossible to satisfy a dependency in
-        an Action.
-    :returns: Iterator of Actions sorted so that all dependent Actions are run
-        before actions which depend on them.
-
-    .. note::
-
-        * The sort is stable but not predictable. The order will be the same
-          as long as the Actions given has not changed but adding or
-          subtracting Actions or dependencies can alter the order a lot.
-
-        * The returned actions do not include ``previously_resolved_actions``.
-          It is up to the caller to processs those before the actions returned
-          by this function.
+    :arg potential_actions: Sequence of Actions which should be run.
+    :arg failed_actions: A set of failed Actions.  resolve_action_order() does not modify this but
+        does expect that its contents will be updated between iterations if there are more
+        failures.
+    :returns: A list of Actions sorted so that all dependent Actions are run before actions which
+        depend on their output.
+    :raises Exception: when it is impossible to satisfy a dependency in an Action.
```

Comment:
Why the docstring here was entirely changed?

Reply:
Not sure, maybe I had an older version of the docstring, I will revert it